source/features/locked-issue.tsx

```diff
@@ -16,21 +18,28 @@ function LockedIndicator(): JSX.Element {
 }
 
 function addLock(element: HTMLElement): void {
-	const classes = (
-		element.closest('.gh-header-sticky')
-			? 'mr-2 '
-			: ''
-	)
-	+ 'mb-2 rgh-locked-issue';
-	element.after(
-		<LockedIndicator className={classes} />,
+	const isIssue = element.getAttribute('data-testid')?.startsWith('issue-metadata');
```

Comment:
This should be the differentiation: hopefully when they enable the view on the PRs it will use the same name/DOM

```suggestion
	const isReactView = element.getAttribute('data-testid')?.startsWith('issue-metadata');
```